upgrading.md

```diff
@@ -0,0 +1,84 @@
+# Upgrading Application Service Adapter
+
+This document describes how to upgrade Application Service Adapter for VMware Tanzu Application Platform.
+
+You can perform a fresh installation of the Application Service Adapter by following the instructions in [Installing Application Service Adapter](install.md).
+
+When upgrading to a new major or minor version of Application Service Adapter, please see that versions' documentation for version-specific configuration and upgrade instructions.
+
+These instructions cover upgrading to a new patch version of the currently installed major minor version of Application Service Adapter.
```

Comment:
> currently installed major minor version of Application Service Adapter
I get what this is saying, but it took me a second to figure out what "major minor version" meant. I'm wondering if we could just simplify this to say "currently installed version", but I get that that is not exactly the same as what this is saying right now. Anyways, I don't really have a better suggestion other than that -- just read oddly to me initially.